apps/expo/src/components/EventTab.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Text, View } from "react-native";
+
+import type { RouterOutputs } from "~/utils/api";
+
+type GetActivityOutput = RouterOutputs["activity"]["getSchedule"][number];
```

Comment:
This might sound a bit weird, but can you create a custom type for the data that you need for the activity. We're currently changing a lot of how schedules / activities are being represented, so it is probably easier to use placeholder here for now. Just addd a TODO to replace the type

Reply:
Yea sure!